src/main/resources/db/changelog/changes/v1.0.0/sql/entity-type-definitions/insert-user-details-definition.sql

```diff
@@ -13,6 +13,110 @@ INSERT INTO entity_type_definition (id, derived_table_name, definition)
+                        }
+                 },
+                {
+                      "name": "cities",
+                      "dataType":{
+                        "dataType":"arrayType",
+                        "itemDataType": {
+                          "dataType": "stringType"
+                        }
+                      },
+                      "labelAlias": "City",
+                      "visibleByDefault": true
+                },
+                {
+                      "name": "regions",
+                      "dataType":{
+                        "dataType":"arrayType",
+                        "itemDataType": {
+                          "dataType": "stringType"
+                        }
+                      },
+                      "labelAlias": "Region",
+                      "visibleByDefault": true
+                },
+                {
+                      "name": "country_ids",
+                      "dataType":{
+                        "dataType":"arrayType",
+                        "itemDataType": {
+                          "dataType": "stringType"
+                        }
+                      },
+                      "labelAlias": "Country ID",
+                      "visibleByDefault": true
+                },
+                {
+                      "name": " postal_codes",
+                      "dataType":{
+                        "dataType":"arrayType",
+                        "itemDataType": {
+                          "dataType": "stringType"
+                        }
+                      },
+                      "labelAlias": "Postal code",
+                      "visibleByDefault": true
+                },
+                {
+                      "name": " address_line1",
+                      "dataType":{
+                        "dataType":"arrayType",
+                        "itemDataType": {
+                          "dataType": "stringType"
+                        }
+                      },
+                      "labelAlias": "Address line1",
+                      "visibleByDefault": true
+                },
+                {
+                      "name": " address_line2",
+                      "dataType":{
+                        "dataType":"arrayType",
+                        "itemDataType": {
+                          "dataType": "stringType"
+                        }
+                      },
+                      "labelAlias": "Address line2",
+                      "visibleByDefault": true
+                },
+                {
+                      "name": " primary_address",
+                      "dataType":{
+                        "dataType":"arrayType",
+                        "itemDataType": {
+                          "dataType": "stringType"
+                        }
+                      },
+                      "labelAlias": "Primary address",
+                      "visibleByDefault": true
```

Comment:
Shouldn't the primary address be a basic string, instead of an array of strings?